### PR TITLE
SAK-30521 Batch webservices for enroll and remove users in a site

### PR DESCRIPTION
**Pull request by ddelblanco**
Improve performance with websevices integrations that need to insert or remove more than one user in a site.

### CODE REVIEW

File: webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import org.sakaiproject.user.api.User;
+import org.sakaiproject.site.api.Site;
+import org.junit.rules.ExpectedException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.lang.RuntimeException;
+
+public class SakaiScriptAddMemberToSiteWithRoleBatchTest extends AbstractCXFTest {
```
Comment (jonespm) [inline webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java]: This file is incorrectly named, failing the build
[ERROR] COMPILATION ERROR : 
[ERROR] /home/travis/build/sakaiproject/sakai/webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java:[37,8] class SakaiScriptAddMemberToSiteWithRoleBatchTest is public, should be declared in a file named SakaiScriptAddMemberToSiteWithRoleBatchTest.java
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:testCompile (default-testCompile) on project sakai-ws: Compilation failure
[ERROR] /home/travis/build/sakaiproject/sakai/webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java:[37,8] class SakaiScriptAddMemberToSiteWithRoleBatchTest is public, should be declared in a file named SakaiScriptAddMemberToSiteWithRoleBatchTest.java

Comment (ddelblanco) [inline webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java]: I don't know why... In my local the file has the Add with "A" and not with "a" but I to amend and commit but it says there are not changes.  I'll take a look in google to view if I need to do something with GIT to send the change in the name of the file.

Comment (ddelblanco) [inline webservices/cxf/src/test/java/org/sakaiproject/webservices/SakaiScriptaddMemberToSiteWithRoleBatchTest.java]: I've removed the file and added again and now it has been updated correctly.  Git magic!!!!

**Comment by ern** [comment]
Looks good and thanks for the tests. Can you add the jira key to the title of this PR?

**Comment by ottenhoff** [comment]
Can you add the JIRA key to the commit?

**Pull request merged**